### PR TITLE
Fix grammar in README

### PR DESCRIPTION
**Pull request by charliewilliams**

### CODE REVIEW

**Comment by alloy** [comment]
Ace, thanks a bunch 👍

**Comment by charliewilliams** [comment]
I have some fixes for guides.cocoapods.org as well, but I can't find the text I want to fix in the repo. Is there a secret to where the examples live?

**Comment by segiddins** [comment]
They're documented from the YARD docs in https://github.com/CocoaPods/Core

**Comment by charliewilliams** [comment]
Thanks!

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Setup
 
-Ideally this website should be ran through a sub-folder of [Strata](https://github.com/CocoaPods/Strata).
+Ideally this website should be run through a sub-folder of [Strata](https://github.com/CocoaPods/Strata).
```
Comment (orta) [inline README.md]: hahaha, my girlfriend gets really annoyed with this one, I do it all the time, it's how we say it in my accent :dancers:

**Pull request merged**